submissions/smillims/cinema-booking/index.html

```diff
@@ -0,0 +1,307 @@
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/media.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header-menu">
+        <input class="input-menu visually-hidden" id="menu" type="checkbox" value="menu" />
+        <label class="label-menu" for="menu">
+          <img src="img/menu.svg" alt="menu" />
+        </label>
+        <aside class="aside-menu">
+          <div class="aside-menu-block-h1">
+            <h1 class="aside-menu-h1">soul</h1>
+          </div>
+          <div class="asime-menu-poster">
+            <img src="img/Soul_(poster).jpg" alt="poster" />
+          </div>
+          <div class="aside-menu-tickets-green tickets">
+            <p class="numbers-tickets">0</p>
+            <img src="img/movie-tickets.svg" alt="ticket" />
+          </div>
+          <div class="aside-menu-tickets-yellow tickets">
+            <p class="numbers-tickets">0</p>
+            <img src="img/movie-tickets.svg" alt="ticket" />
+          </div>
+          <div class="aside-menu-tickets-blue tickets">
+            <p class="numbers-tickets">0</p>
+            <img src="img/movie-tickets.svg" alt="ticket" />
+          </div>
+          <button class="aside-menu-button" type="button">Buy</button>
+        </aside>
+      </div>
+    </header>
+    <div class="container">
+      <main class="main">
+        <img class="main-poster" src="img/soul.jpg" alt="poster" />
+        <form class="main-form">
+          <div class="block-date">
+            <p class="form-date">Date:</p>
+            <select class="choose-date" id="date" name="date">
+              <option disabled>Choose date:</option>
+              <option value="20.02">20.02</option>
+              <option value="21.02">21.02</option>
+              <option value="22.02">22.02</option>
+              <option value="23.02">23.02</option>
+            </select>
+          </div>
+          <div class="block-time">
+            <p class="form-time">Time:</p>
+            <select class="choose-time" id="time" name="time">
+              <option disabled>Choose time:</option>
+              <option value="10:00">10:00</option>
+              <option value="15:00">15:00</option>
+              <option value="18:00">18:00</option>
+              <option value="21:00">21:00</option>
+            </select>
+          </div>
+        </form>
+      </main>
+
+      <section class="seating">
+        <div class="seating-screen">
+          <svg width="629" height="27" viewBox="0 0 629 27" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M1 26L19.4899 23.2417C216.102 -6.08887 415.988 -5.94341 612.557 23.6733L628 26" stroke="black" />
+          </svg>
+        </div>
+        <form class="form-seating">
+          <div class="seating-container">
+            <div class="seating-green">
+              <input class="seating-place-green visually-hidden" id="green-1" type="checkbox" value="green" />
```

Comment:
You have a lot of inputs with the same value. How you will know which checkbox you choose?)